packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -365,7 +365,7 @@ class SemanticsData extends Diagnosticable {
       scrollExtentMax,
       scrollExtentMin,
       transform,
-      customSemanticsActionIds,
+      ui.hashList(customSemanticsActionIds),
```

Comment:
Just curious: why the hash list?

Reply:
we have an assert in `ui.hashValues` that fails for iterables - not sure how it didn't catch this before, or if it is new.